install/running_install.adoc

```diff
@@ -483,15 +483,15 @@ The following table lists the playbooks in the order that they must run:
 |Web Console Install
 |*_{pb-prefix}playbooks/openshift-web-console/config.yml_*
 
+|Admin Console Install
+|*_{pb-prefix}playbooks/openshift-console/config.yml_*
+
```

Comment:
Admin Console Install looks good to me, sorry for late reply

Reply:
Thank you!